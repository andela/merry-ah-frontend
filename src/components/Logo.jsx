import React, { Component } from 'react';
import { NavLink } from 'react-router-dom';

/**
 * Logo Component
 */
export default class Logo extends Component {
  /**
 * @param {function} render
 *  @returns {JSX} jsx
 */
  render() {
    return (
      <NavLink
      className="navbar-brand col-lg-2 logo pull-left w-50"
      to="/" id="brand-logo">
        <img alt={'logo'} src={require('../assets/images/artcave-logo.png')}
             className='w-80' />
      </NavLink>
    // <NavLink
    // className="navbar-brand col-lg-2 logo pull-left w-50"
    // to="/" id="brand-logo">
    //   <img alt={'logo'}
    //     src='../assets/images/artcave-logo.png' className='w-50' />
    //   </NavLink>
    );
  }
}
