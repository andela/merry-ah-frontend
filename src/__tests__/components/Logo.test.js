/* eslint-disable import/no-unresolved */
import React from 'react';
import { shallow } from 'enzyme';
import Logo from '../../components/Logo';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Logo />);
});
describe('Logo Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
