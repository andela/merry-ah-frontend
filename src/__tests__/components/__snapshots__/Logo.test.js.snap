// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should match snapshot 1`] = `
<NavLink
  activeClassName="active"
  aria-current="page"
  className="navbar-brand col-lg-2 logo pull-left w-50 p-0"
  id="brand-logo"
  to="/"
>
  <img
    alt="logo"
    className="w-25"
    src="../assets/images/artcave-logo.png"
  />
   artCave
</NavLink>
`;
